python/pdo/service_client/enclave.py

```diff
@@ -39,6 +49,10 @@ class EnclaveServiceClient(GenericServiceClient) :
 
     def __init__(self, url) :
         super().__init__(url)
+        self.session = requests.Session()
```

Comment:
1. how important is to close the session before client process exits? 

2. Should the client check if the session is alive before every request? I guess my question really is if the server can drop a session due to client inactivity?

Reply:
no close required.
session designed to restart dropped connection.